### PR TITLE
update on Portfolio page

### PR DESCRIPTION
**Pull request by emmanuelsteven**
added more content to the page.
added the project sections.
added background images 
manipulated a few list items listed some of the courses i would be exposed to in the course of the program.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -109,3 +129,199 @@ p {
 .logo2 {
   padding: 20px 20px 20px 0;
 }
+
+.header2 {
```
Comment (topeogunleye) [inline style.css:133]: - [ ] As required, I kindly suggest that you ensure the main container of your works section has a display of grid.

File: style.css [hunk 2]
```diff
@@ -109,3 +129,199 @@ p {
...
+
+.nx {
+  display: flex;
+  gap: 12px;
+  flex-direction: row;
+  list-style-type: none;
+  padding-left: 24px;
+}
+
+button {
```
Comment (topeogunleye) [inline style.css:198]: - [ ] I kindly suggest that you add the correct button interactions including hover and active (when it's pressed), as shown in the Figma template below:

image

the button on the pressed state has the `background: #E05304` and on hover has the `background: #FFF5E1`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #E05304;
}
```
**for hover state**
```
button:hover {
    background-color: #FFF5E1;
}
```

File: index.html [hunk 3]
```diff
@@ -51,6 +51,147 @@ <h2>I am a Software Developer</h2>
...
+        <div class="header3">
+          <h2 class="hf">About Me</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button class="button3">Get My Resume</button>
+         <div class="bckimg"> <img src="img2/Group (3).png" alt=""></div>
```
Comment (topeogunleye) [inline index.html]: - [ ] I kindly suggest that you use the `background-image` property through your `style.css` to apply your background images.

Kindly check the video below to know how to download your background image:


https://user-images.githubusercontent.com/52934128/219375621-71d4d0ca-fa5e-4bd8-9732-22c25ffc63ae.mp4

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @emmanuelsteven, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @emmanuelsteven 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**